core/ir/instr_api.h

```diff
@@ -310,6 +310,13 @@ struct _instr_t {
     byte num_dsts;
     byte num_srcs;
 
+    /** This field assumes values of type #dr_isa_mode_t.
```

Comment:
nit: "assumes" can be confused with assumptions: better "holds" or "stores".

Reply:
Oh right!
Done.
Using "holds".